src/spacewar/ui/widgets/lcars.cljc

```diff
@@ -29,6 +29,15 @@
        :c2 [(+ x w (- uic/stringer-width)) (+ y uic/banner-width)]
        :label-position [(+ x 10) (+ y 10)]})))
 
+(defn quadratic-vertex [x1 y1 cx cy x3 y3]
+  (q/bezier-vertex
+    (+ x1 (* (/ 2 3) (- cx x1)))
+    (+ y1 (* (/ 2 3) (- cy y1)))
+    (+ x3 (* (/ 2 3) (- cx x3)))
+    (+ y3 (* (/ 2 3) (- cy y3)))
+    x3
+    y3))
+
```

Comment:
This `quadratic-vertex` polyfill makes up for it being missing in the ClojureScript Quil API. Its implementation is a ClojureScript port of the implementation on the Clojure side (which calls `bezier-vertex`).